exporters/otlp/all/build.gradle.kts

```diff
@@ -41,6 +39,45 @@ val testJavaVersion: String? by project
 
 testing {
   suites {
+    listOf(
+      "LATEST",
+      "4.11.0"
```

Comment:
We would enumerate the list here. "LATEST" is a special keyword which just uses the dependency from `:dependencyConfiguration`.